lib/bootloader_setup.pm

```diff
@@ -308,7 +309,7 @@ sub bootmenu_type_console_params {
 
     # See bsc#1011815, last console set as boot parameter is linked to /dev/console
     # and doesn't work if set to serial device. Don't want this on ipmi backend.
-    type_string_very_slow "console=tty " unless check_var('BACKEND', 'ipmi');
+    type_string_very_slow "console=tty " unless (check_var('BACKEND', 'ipmi') || check_var('BACKEND', 'spvm'));
```

Comment:
didn't we want to use "is_remote_backend" or something? We have that in lib/main_common.pm by now

Reply:
done